src/app/state/card/card.selectors.ts

```diff
@@ -68,9 +68,45 @@ export const selectCardsPageInfo = createSelector(
     pageIndex: cardState.pageIndex,
   }),
 );
+export const selectCardsSorted = createSelector(
+  selectAllCards,
+  selectCardState,
+  (allCards, cardState) => {
+    const getLastName = (obj: Card) => obj?.name?.split(' ')?.pop() || obj.name;
```

Comment:
Diese Logik ist nicht ganz kugelsicher (z.B. bei "Sarah Moy de Vitry"). Wenn wir sauber nach Nachnamen sortieren wollen, müssten wir das DB Modell erweitern, oder?

Reply:
Unsere Überlegung war dass ansonsten Kürzel und Name keine großen Unterschied hat...aber wir können auch einfach nach String sortieren?

Reply:
Mir persönlich wäre das lieber als eine Sortierung nach Nachname, die nicht in allen Fällen stimmt. Aber up to you!

Reply:
Okay 👍 hab's wieder zurück geändert